data/brands/amenity/fast_food.json

```
@@ -9240,6 +9240,18 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Shah's Halal Food",
+      "locationSet": {
+        "include": ["ca", "gb", "se", "us"]
+      },
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Shah's Halal Food",
+        "brand:wikidata": "",
+        "name": "Shah's Halal Food",
```

I assume we want to add this (dunno how to easily check if it's yes or only):
```suggestion
        "diet:halal": "yes",
        "name": "Shah's Halal Food",
```